building/export_plugin_from_eclipse.adoc

```diff
@@ -0,0 +1,37 @@
+= How to Export a plugin from Eclipse
+
+Exporting a plugin from inside eclipse is the 'quick and dirty' method to get an output binary for quick local tests. It does not generate an update site, and so this mechanism is only really suitable for local smoke-testing. 
```

Comment:
okey - may I suggest we just drop that and say to use the quick and correct mvn verify and pick it up from site/target ?